src/Pages/adminEdit/AdminCreate.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState } from 'react';
+import FoodMenu from './FoodMenu';
+import { INPUT_VALUES, CLOSED_DAY, CATEGORIES } from './adminEditData';
+import './AdminEdit.scss';
+
+const AdminCreate = () => {
+  const accesToken = localStorage.getItem('TOKEN');
+
+  const [input, setInput] = useState({
+    name: '',
+    description: '',
+    address: '',
+    tel: '',
+    open_time: '',
+    closed_time: '',
+    closed_day_id: '',
+    price_range: ' ',
+    category_id: '',
+  });
+
+  const [imageInput, setImageInput] = useState();
+  const [dayNum, setDayNum] = useState(0);
+  const [menuId, setMenuId] = useState(1);
+  const [menus, setMenus] = useState([]);
+  const [menuInput, setMenuInput] = useState({
+    id: menuId,
+    name: '',
+    price: '',
+  });
+
+  const saveInput = e => {
+    const { name, value } = e.target;
+    setInput({ ...input, [name]: value });
```

Comment:
```suggestion
    setInput(prev => ({ ...prev, [name]: value }));
```
함수형 업데이트로 구현하는 방법이 더욱 안전한 방법입니다!